charts/linkerd2/templates/_config.tpl

```diff
@@ -0,0 +1,80 @@
+{{- define "linkerd.configs.global" -}}
+{
+  "linkerdNamespace": "{{.Namespace}}",
+  "cniEnabled": false,
+  "version": "{{.LinkerdVersion}}",
+  "identityContext":{
+    "trustDomain": "{{.Identity.TrustDomain}}",
+    "trustAnchorsPem": "{{.Identity.TrustAnchorsPEM | replace "\n" "\\n"}}",
```

Comment:
`Identity.TrustAnchorsPEM` is required during injection if identity is enabled, so you might wanna add a `require` statement here in such case.